src/components/MakeWidget/subcomponents/ExpirySelector/ExpirySelector.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from "react";
+
+import { Dropdown, SelectOption } from "../../../Dropdown/Dropdown";
+import { Wrapper, Title, Input } from "./ExpirySelector.styles";
+
+const MS_PER_MINUTE = 60000;
+
+const options = [
+  {
+    label: "minutes",
+    value: `${MS_PER_MINUTE}`,
+  },
+  {
+    label: "hours",
+    value: `${60 * MS_PER_MINUTE}`,
+  },
+  {
+    label: "days",
+    value: `${24 * 60 * MS_PER_MINUTE}`,
+  },
+  {
+    label: "weeks",
+    value: `${7 * 24 * 60 * MS_PER_MINUTE}`,
+  },
+];
+
+const floatRegExp = new RegExp("^([0-9])*$");
+
+export type ExpirySelectorProps = {
+  updateExpiry: (msToExpiry: number) => void;
```

Comment:
```suggestion
    onChange: (expiryDate: number) => void;
```
Value returned should be current date + expiry time